app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -138,7 +138,7 @@ def record_class
   end
 
   def retrieve_provider_regions
-    managers = ManageIQ::Providers::CloudManager.supported_subclasses.select(&:supports_regions?)
+    managers = ManageIQ::Providers::CloudManager.permitted_subclasses.select(&:supports_regions?)
```

Comment:
NOTE after https://github.com/ManageIQ/manageiq/pull/21553 this can become `ManageIQ::Providers::CloudManager.supporting(:regions)`